mypy/nodes.py

```diff
@@ -1649,6 +1859,46 @@ def __str__(self) -> str:
                             ('Names', sorted(self.names.keys()))],
                            'TypeInfo')
 
+    def serialize(self) -> Union[str, JsonDict]:
+        data = {'.class': 'TypeInfo',
+                'fullname': self.fullname(),
+                'alt_fullname': self.alt_fullname,
+                'subtypes': [t.serialize() for t in self.subtypes],
```

Comment:
We shouldn't serialize `TypeInfo`s for subtypes but include references. However, it seems that this isn't actually used, so we may be able to just leave it out and remove the `subtypes` attribute.